lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -74,7 +74,7 @@ def install!
 
             Pod::Generator::AppTargetHelper.add_app_host_main_file(project, app_host_target, platform_name, @group, app_target_label) if add_main
             Pod::Generator::AppTargetHelper.add_launchscreen_storyboard(project, app_host_target, @group, deployment_target, app_target_label) if platform == :ios
-            additional_entries = platform == :ios ? ADDITIONAL_IOS_INFO_PLIST_ENTRIES : {}
+            additional_entries = ALLOW_INSECURE_LOADS_FOR_TESTS.merge(platform == :ios ? ADDITIONAL_IOS_INFO_PLIST_ENTRIES : {})
```

Comment:
is there a test that covers the additional entries already? maybe integration spec? should be sufficient

Reply:
Yes the integration test covers it.